worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -580,19 +588,22 @@ class FetchHandler(
         streamChunkSlice.chunkIndex,
         streamChunkSlice.offset,
         streamChunkSlice.len)
+      val bufSize = buf.size()
       chunkStreamManager.chunkBeingSent(streamChunkSlice.streamId)
       client.getChannel.writeAndFlush(new ChunkFetchSuccess(streamChunkSlice, buf))
         .addListener(new GenericFutureListener[Future[_ >: Void]] {
           override def operationComplete(future: Future[_ >: Void]): Unit = {
             if (future.isSuccess) {
               if (log.isDebugEnabled) {
                 logDebug(
-                  s"Sending ChunkFetchSuccess to $remoteAddr succeeded, chunk $streamChunkSlice")
+                  s"Sending ChunkFetchSuccess to $remoteAddr succeeded," +
+                    s" chunk $streamChunkSlice, buf size: $bufSize")
```

Comment:
minor change for real chunk size, for `streamChunkSlice`, its length is always Int.MaxValue